packages/video_player/video_player_platform_interface/lib/video_player_platform_interface.dart

```diff
@@ -350,6 +350,11 @@ class DurationRange {
 
 /// [VideoPlayerOptions] can be optionally used to set additional player settings
 class VideoPlayerOptions {
+  /// Set this to true to let plugin manage play/pause with app lifecycle changes,
+  /// Set this to false to keep playing video in background, when app goes in background
+  /// The default value is true
+  final bool observeAppLifecycle;
```

Comment:
I think it would make more sense to call this `pauseInBackground` or similar; observing the app lifecycle is how it's implemented, not the actual user intent of the option.

Reply:
maybe `needBackgroundPlayback`?

Reply:
I'm not sure about "need", but `allowBackgroundPlayback` sounds good to me.

Reply:
fixed